src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -144,6 +144,23 @@
   </div>
 {% endmacro %}
 
+{% macro claimRejectedDefendantDefenceFreeMediationYESForClaimantDashboard(claim) %}
+  {{ t('You’ve rejected the defendant’s response.') }}
+
+  {% if claim.claimantResponse.freeMediation === FreeMediationOption.YES %}
+    <p>{{ t('You’ve both agreed to try mediation.') }}</p>
+  {% endif %}
+{% endmacro %}
+
+{% macro claimRejectedDefendantDefenceFreeMediationYESForClaimantDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('You’ve rejected the defendant’s response.') }}</h2>
+    <p>{{ t('You’ve both agreed to try mediation. The Small Claims Mediation Service will contact you to arrange an appointment.') }}</p>
+    {{ externalLink('Find out how mediation works', 'https://www.gov.uk/government/publications/small-claims-mediation-service-ex730') }}
+    <p>{{ _downloadResponseLink(claim, 'Download your response') }}</p>
```

Comment:
Isn't this the defendant's response link? (If so, check the rest of this file!)

Reply:
updated one of the others that was wrong.